src/main/java/kenigsberg/wordledictionary/WordleGameMain.java

```diff
@@ -0,0 +1,19 @@
+package kenigsberg.wordledictionary;
+
+import org.mockito.Mockito;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.doReturn;
+
+public class WordleGameMain {
+    public static void main(String[] args) {
+        WordleDictionary wordleDictionary = Mockito.mock(WordleDictionary.class);
+        ArrayList<String> words = new ArrayList<>(List.of("APPLE"));
+        doReturn(words).when(wordleDictionary).getList();
```

Comment:
You only want to use mocks in tests. We are writing the game, you want the full dictionary.